src/main/java/org/dinosaur/foodbowl/domain/comment/application/CommentService.java

```diff
@@ -40,4 +42,19 @@ public Long save(Long memberId, CommentCreateRequest commentCreateRequest) {
         Comment savedComment = commentRepository.save(comment);
         return savedComment.getId();
     }
+
+    @Transactional
+    public void updateComment(Long commentId, Long memberId, CommentUpdateRequest commentUpdateRequest) {
+        Comment comment = commentRepository.findById(commentId)
+                .orElseThrow(() -> new FoodbowlException(COMMENT_NOT_FOUND));
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new FoodbowlException(MEMBER_NOT_FOUND));
+
+        if (comment.isNotBelongTo(member)) {
+            throw new FoodbowlException(MEMBER_UNAUTHORIZED);
+        }
```

Comment:
```suggestion
        if (!comment.isAuthor(member)) {
            throw new FoodbowlException(MEMBER_UNAUTHORIZED);
        }
```
도메인의 의미를 살려서 작성자인지 확인하는 메서드명을 사용해보는건 어떨까요?

Reply:
오 저는 `!comment.doWrite(member)` 생각했는데 isAuthor가 낫겠네요

Reply:
`comment.isAuthor(member)`를 처음에 먼저 선택했었는데 `댓글이 회원의 작성자라면`으로 해석이 되는데 좀 어색하지 않나요?

`comment.isBelongTo(member)`를 `댓글이 회원의 소유가 아니면`으로 해석되는 것 같습니다 !

Reply:
말씀해주신 것처럼 `isAuthor`은 `comment`라는 도메인의 관점에서 봤을 때 어울리지 않는 메서드명이네요 😅

그렇다면 `isWrittenBy`와 같은 메서드명은 어떨까요? 속하는 것보다는 작성되었다는 의미가 포함되면 좋을 것 같습니다~~

Reply:
오..... 정말 좋은데요?
제가 원하던 작명입니다...